package.json

```diff
@@ -75,7 +75,7 @@
     "chosen-js": "1.8.2",
     "classnames": "2.2.5",
     "codecov": "3.0.0",
-    "coveo-styleguide": "~2.9.0",
+    "coveo-styleguide": "2.9.x",
```

Comment:
That's my fault, I merged a PR from GreenKeeper. I guess next time will just have to update manually and then let the PR close itself.